clients/admin-ui/src/features/configure-consent/ConsentManagementTable.tsx

```diff
@@ -207,17 +207,18 @@ export const ConsentManagementTable = () => {
         ),
         header: (props) => <DefaultHeaderCell value="Categories" {...props} />,
       }),
-      columnHelper.accessor((row) => row.cookies, {
-        id: "cookies",
-        cell: (props) => (
-          <BadgeCellCount
-            plSuffix="cookies"
-            singSuffix="cookie"
-            count={props.getValue()}
-          />
-        ),
-        header: (props) => <DefaultHeaderCell value="Cookies" {...props} />,
-      }),
+      // TODO: uncomment when cookies are returned from the API
```

Comment:
"the API" meaning ours right? We might have already gone over that and I am just forgetting but which endpoint?